parsimonious/grammar.py

```diff
@@ -68,18 +68,10 @@ def __init__(self, rules='', **more_rules):
             k: (expression(v, k, self) if isfunction(v) or ismethod(v) else v)
             for k, v in iteritems(more_rules)}
 
-        self._expressions, first = self._expressions_from_rules(rules, decorated_custom_rules)
+        expressions, first = self._expressions_from_rules(rules, decorated_custom_rules)
```

Comment:
I guess we might as well either change this to `exprs` to match our spelling elsewhere in this class or change the other places (if you have a strong opinion).